From 317467a5565532d16fb489edff4515dc441ba888 Mon Sep 17 00:00:00 2001
From: aksunknk <aksaks.suak@gmail.com>
Date: Tue, 29 Apr 2025 17:03:57 +0900
Subject: [PATCH] no message

---
 withham_project/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/withham_project/settings.py b/withham_project/settings.py
index 440b7ca..64e8038 100644
--- a/withham_project/settings.py
+++ b/withham_project/settings.py
@@ -31,7 +31,7 @@ if dotenv_path.exists():
 # ここにはDjangoが自動生成した実際のシークレットキーが入ります。
 # 絶対に公開しないでください。
 # ★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★★
-SECRET_KEY = os.environ.get('SESECRET_CRET_KEY')
+SECRET_KEY = os.environ.get('SECRET_KEY')
 
 if not SECRET_KEY:
     # raise ValueError("SECRET_KEY environment variable not set.") # エラーにする場合
-- 
2.49.0.windows.1

